src/agent/vsoworker.ts

```diff
@@ -109,7 +109,7 @@ export function run(msg: cm.IWorkerMessage, consoleOutput: boolean,
         trace.state('msg:', msg);
 
         var agentUrl = hostContext.config.settings.serverUrl;
-        var taskUrl = job.environment.variables[cm.vars.systemTfCollectionUri];
+        var taskUrl = job.environment.systemConnection.url;
```

Comment:
With s89 deployment the variable will have tfs url and not the service speicifc url. This will not impact build but RM. All the calls to DT sdk from RM would fail.